app/views/users/twitter_search.html.erb

```diff
@@ -0,0 +1,16 @@
+<h1> Search Results </h1>
+<div class="col-sm-6">
+  <% @search_results.each do |search| %>
+  <a href="/twitter_results/<%= search.screen_name %>" >
```

Comment:
Since you already have twitter_search_user path set up as a helper in your routes, you can just do this:
```ruby
<%= link_to twitter_search_user(user.screen_name) %>
```